organisations/service.go

```diff
@@ -135,14 +135,36 @@ func (cd service) Write(thing interface{}) error {
 		}
 		queries = append(queries, parentQuery)
 	}
-	return cd.cypherRunner.CypherBatch(queries)
+
+	return processCypherBatchWrite(cd, queries)
+}
+
+func processCypherBatchWrite(cd service, queries []*neoism.CypherQuery) error {
```

Comment:
If we end up needing this elsewhere, we could create a cypherRunner in neoutils that does this error extraction and maps back to the right kind of error in there? Maybe worth doing now, and it's probably useful outside the context of the base app, so maybe put a new cypherrunner and associated ConflictError type in neoutils.  What do you think?